plutus-script-reexecutor.cabal

```diff
@@ -10,7 +10,7 @@ library
   build-depends:
     , base                         >=4.9   && <5
 
-executable exec
+executable exec 
```

Comment:
This is me being pedantic, but can we remove this?
I have a save hook that removes any trailing spaces.